apps/webapp/src/state/send.test.ts

```diff
@@ -90,9 +88,9 @@ describe('Send Slice', () => {
 
     test('validate high enough amount validates', () => {
       const assetBalance = new Amount({ hi: 1n });
-      const state = produce(selectionExample, draft => {
-        draft.value.valueView.value!.amount = assetBalance;
-      });
```

Comment:
produce can only be called on things that are draftable: plain objects, arrays, Map, Set or classes that are marked with '[immerable]: true'
we can't use `immer` for `BalancesResponse`, so we modify it directly